js/adapt-contrib-slider.js

```diff
@@ -370,15 +370,24 @@ define([
             var $markers = this.$('.slider-markers').empty();
             if (this.model.get('_showScale') === false) {
                 $markers.eq(0).css({display: 'none'});
-                this.$('.slider-scale-numbers').eq(0).css(
+                this.$('.slider-scale-numbers *:not(".slider-scale-marker")').css(
                     this.model.get('_showScaleIndicator') ? {visibility: 'hidden'} : {display: 'none'}
                 );
```

Comment:
if you `return;` here you can lose the `else` nesting and it'll all look a bit tidier